share/spice/bike_sharing/citi_bike_nyc/bike_sharing_citi_bike_nyc.js

```diff
@@ -0,0 +1,79 @@
+(function (env) {
+    "use strict";
+
+    env.ddg_spice_bike_sharing_citi_bike_nyc = function(api_result) {
+        if (!api_result || !api_result.stationBeanList) {
+            return Spice.failed('citibikenyc');
+        }
+
+        DDG.require('moment.js', function() {
+        DDG.require('maps', function() {
+            var script = $('[src*="/js/spice/bike_sharing/citi_bike_nyc/"]')[0];
+            var source = $(script).attr("src");
+            var query = source.match(/citi_bike_nyc\/([^\/]+)/)[1];
+
+            if (query) {
+                var references = {
+                    'brooklyn': L.latLng(40.645244, -73.9449975),
```

Comment:
Seems like the return from this is just a standard object with a `Lat: ...` and `Lon: ...` -- do we need to use Leaflet here?